src/span.js

```diff
@@ -171,39 +171,54 @@ export default class Span {
     }
 
     /**
-     * Explicitly create a log record associated with the span.
+     * Add a log record to this Span, optionally at a user-provided timestamp.
      *
-     * @param {object} fields - object containing the log record properties
-     * @param {number} [fields.timestamp] - optional field specifying the
-     *        timestamp in milliseconds as a Unix timestamp. Fractional values
-     *        are allowed so that timestamps with sub-millisecond accuracy
-     *        can be represented. If not specified, the implementation is
-     *        expected to use it's notion of the current time of the call.
-     * @param {string} [fields.event] - the event name
-     * @param {object} [fields.payload] - an arbitrary structured payload. It is
-     *        implementation-dependent how this will be processed.
+     * For example:
+     *
+     *     span.log({
+     *         size: rpc.size(),  // numeric value
+     *         URI: rpc.URI(),  // string value
+     *         payload: rpc.payload(),  // Object value
+     *         "keys can be arbitrary strings": rpc.foo(),
+     *     });
+     *
+     *     span.log({
+     *         "error.description": error.description(),  // numeric value
+     *     }, error.timestampMillis());
```

Comment:
This is/was just an imaginary example object... and I'm annoyed that I just merged this since the preceding line has an incorrect "numeric value" comment. I'll fix that in master.